server/tests/users.test.js

```diff
@@ -1,13 +1,29 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
+import faker from 'faker';
+import models from '@models';
 import app from '../app';
+import generateToken from './factory/user-factory';
 
+let userToken;
+const { User } = models;
 chai.use(chaiHttp);
 const { expect } = chai;
 const invalidToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjZjZDAwNDBmLTI3MDktNGU0Yi05YjU2LWYzZDk3MmRhNjk4OTg5IiwiZW1haWwiOiJqdXN0c2luZUBzbnF3c3QuY29tIiwiaWF0IjoxNTYwMjA3NTAyLCJleHAiOjE1NjAyOTM5MDJ9.FpXu8SrboezKr57MNcrEA_pGhsMRm0G5ptUGqQje12I';
-let userToken;
+
+let authToken;
 
 describe('TESTS TO SIGNUP A USER', () => {
+  before(async () => {
+    const newUser = await User.create({
```

Comment:
move user creation into the factory (and call the needed functions from here)

so
```javascript
User.create({
...
```
would be located in factory as a function called `createTestUser`